### PR TITLE
NXP-25040: enable Tomcat's rewrite valve

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

**Comment by ataillefer** [comment]
:+1:

File: nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common/META-INF/templates/rewrite.config [hunk 1]
```diff
@@ -0,0 +1 @@
+%{RULE}%
```
Comment (efge) [inline nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common/META-INF/templates/rewrite.config]: Please add in the JIRA ticket description an example of how to use this within a `deployment-fragment.xml`.

File: nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common/META-INF/nuxeo-preprocessor.xml.nxftl [hunk 2]
```diff
@@ -18,6 +18,9 @@
   <template name="jaxws" src="META-INF/templates/sun-jaxws.xml"
     installPath="nuxeo.war/WEB-INF/sun-jaxws.xml" required="false" />
 
+  <template name="rewrite" src="META-INF/templates/rewrite.config"
+    installPath="nuxeo.war/WEB-INF/rewrite.config" required="true" />
```
Comment (efge) [inline nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common/META-INF/nuxeo-preprocessor.xml.nxftl]: Why `required="true"` ?

Comment (nelsonsilva) [reply nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common/META-INF/nuxeo-preprocessor.xml.nxftl]: For no reason. Thx 👍

**Comment by efge** [comment]
👍

**Pull request merged**